src/providers/wfs/qgswfssourceselect.h

```diff
@@ -103,7 +103,8 @@ class QgsWFSSourceSelect: public QgsAbstractDataSourceWidget, private Ui::QgsWFS
 
     void populateConnectionList();
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void on_buttonBox_helpRequested() { QgsHelp::openHelp( QStringLiteral( "working_with_ogc/ogc_client_support.html" ) ); }
+
 
```

Comment:
Can you fix this?

In general it's safer to switch to `private slots:` and `connect` instead of these "magic method names".